app/cafe.py

```diff
@@ -0,0 +1,24 @@
+import datetime
+
+from app.errors import (
+    NotVaccinatedError,
+    OutdatedVaccineError,
+    NotWearingMaskError
+)
+
+
+class Cafe:
+    def __init__(self, name: str) -> None:
+        self.name = name
+
+    def visit_cafe(self, visitor: dict) -> str | None:
+        if visitor.get("vaccine") is None:
+            raise NotVaccinatedError("NotVaccinatedError")
```

Comment:
if there is vaccine but it's an empty dict, then the condition below will raise a key error, let's handle this case as well